src/app/components/analogues/analogue-detail/analogue-detail.component.html

```diff
@@ -0,0 +1,94 @@
+<div class="source-container" *ngIf="headVisible">
+        
+    <div class="card-body app-detail-content source-detail-content" *ngIf="analogue.content.length > 0">
+        <span class="btn btn-sm transparent source-quote-button">☚</span>
```

Comment:
Can you use a specific icon?
EVT use his own font-icon

Reply:
The icon has been removed and it's not displayed, it's used as a signpost at the moment